lib/PuppeteerSharp.TestServer/PuppeteerSharp.TestServer.csproj

```diff
@@ -1,13 +1,11 @@
-<Project Sdk="Microsoft.NET.Sdk.Web">
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">

```

Comment:
Let's undo this change. It makes no difference

Reply:
Yeah, it's Rider that makes some formatting ...

Reply:
I think it's better now

Reply:
I can't see the difference on this line, maybe the BOM or different carriage return on that line ...